musit-service/src/main/scala/no/uio/musit/security/dataporten/DataportenAuthenticator.scala

```diff
@@ -57,19 +71,12 @@ class DataportenAuthenticator @Inject()(
   private type DataportenToken = BearerToken
   private type AuthResponse    = Future[Either[Result, UserSession]]
 
-  private val defaultSessionTimeout = (2 hours).toMillis
```

Comment:
The default session timeout of 2 hours is quite important. But I can't see that it's being taken care of in the changes you've made?